platform-tests/src/test/java/org/junit/platform/suite/commons/SuiteLauncherDiscoveryRequestBuilderTests.java

```diff
@@ -214,6 +217,88 @@ class Suite {
 		assertEquals(TestCase.class, exactlyOne(selectors).getJavaClass());
 	}
 
+	@Test
+	void selectMethodsOnlyByName() {
```

Comment:
It is not possible to select methods only by name. It would be prudent to build a snapshot version of your branch and try it out in a test project. You should try to use (and break) the code you're writing.